### ISSUE TITLE
[GPU renderer] Broken transparency

### ISSUE THREAD

**Issue by PrgmrAtHrt** [luciusDXL/TheForceEngine#315]
![bugged gfx](https://github.com/luciusDXL/TheForceEngine/assets/38510213/7ef36052-bce6-440e-9ace-e8f534da2779)

Intel i5-3320m + Intel HD 4000 Graphics

**Comment by PrgmrAtHrt** [comment]
There is no update. This chipset has been sunsetted.

**Comment by 000MDK** [comment]
> There is no update. This chipset has been sunsetted.

https://www.intel.com/content/www/us/en/support/articles/000057520/graphics.html

There is an older driver that supports DX12 and the newer one, that supports HVCI but dropped DX12.
I use the new driver with Windows 11 with i5/HD4600 and TheForceEngine works as it should. (I use max allocated mem in BIOS FYI)

Have you checked your gamedata for corruption (redownload with steam/gog)?
Also run ```sfc /scannow``` in admin-Terminal/cmd to be sure your systemfiles are okay.

**Comment by PrgmrAtHrt** [comment]
The driver you linked is for a generation newer than my chipset and is not compatible. My gamedata and system files are fine.

**Comment by PrgmrAtHrt** [comment]
> > There is no update. This chipset has been sunsetted.
> 
> https://www.intel.com/content/www/us/en/support/articles/000057520/graphics.html
> 
> There is an older driver that supports DX12 and the newer one, that supports HVCI but dropped DX12. I use the new driver with Windows 11 with i5/HD4600 and TheForceEngine works as it should. (I use max allocated mem in BIOS FYI)
> 
> Have you checked your gamedata for corruption (redownload with steam/gog)? Also run `sfc /scannow` in admin-Terminal/cmd to be sure your systemfiles are okay.

The driver you linked is for a generation newer than my chipset and is not compatible. My gamedata and system files are fine.

**Comment by PrgmrAtHrt** [comment]
> Looks like an _8-bit color mode_ issue.
> 
> @PrgmrAtHrt Try setting “Color mode” in “Graphics” to `True Color`.
> 
> @luciusDXL Depending on what the _8-bit color modes_ are supposed to do, setting `True Color` may not be the solution but a workaround only.

Yes, true color does indeed work fine for me. However, this is a workaround; not a solution.

**Comment by luciusDXL** [comment]
This is likely a rounding issue when looking up palettized colors. I will look into implementing a work around at some point. Good to know that true color works. :)
